src/pages/HomePage/HomePage.styles.js

```diff
@@ -0,0 +1,151 @@
+import styled from 'styled-components';
+import mediaQuery from '../../utils/mediaQuery';
+
+export const BannerWrapper = styled.div`
+  background: ;
+  height: 166px;
```

Comment:
missing value

Reply:
Removed background style, as background color is common for all the pages it is better to add in main component.